Xamarin.Forms.Platform.Android/FastRenderers/AccessibilityProvider.cs

```diff
@@ -145,8 +145,7 @@ bool SetHint(string hint = GetFromElement)
 			string value = hint;
 			if (value == GetFromElement)
 			{
-				value = string.Join(". ", (string)Element.GetValue(Accessibility.NameProperty),
-					(string)Element.GetValue(Accessibility.HintProperty));
+				value = string.Join((String.IsNullOrWhiteSpace((string)(Element.GetValue(Accessibility.NameProperty))) || String.IsNullOrWhiteSpace((string)(Element.GetValue(Accessibility.HintProperty)))) ? "" : ". ", (string)Element.GetValue(Accessibility.NameProperty), (string)Element.GetValue(Accessibility.HintProperty));
```

Comment:
Agree, some other thoughts on the setHint method.

line 146 if (value == GetFromElement)
My eyes tells me this will allways be true since value is just a reference to hint wich  is allways pointing to GetFromElement. (I think there is a reason for this, but i just dont see it)

line 151 textView.Hint = !string.IsNullOrWhiteSpace(value) ? value : _defaultHint;
since value never will be null due to the join ". " the hint will allways be set to ". " if no Accessibility content is set.

wouldnt it be more efficient just to check if the accessibility properties has values, if not early return?